page-index/page-index.js

```diff
@@ -439,53 +439,81 @@ const pageIndex = [
         href: "tutorials",
         children: [
             {
-                Title: "Get started: Hello, Timescale!",
-                type: PAGE,
-                href: "tutorial-hello-timescale",
-                children: []
-            }, {
-                Title: "Time-series forecasting",
-                type: PAGE,
-                href: "tutorial-forecasting",
-                children: []
-            }, {
-                Title: "Replication",
-                type: PAGE,
-                href: "replication",
-                children: []
-            }, {
-                Title: "Continuous aggregates",
-                type: PAGE,
-                href: "continuous-aggs-tutorial",
-                children: []
-            }, {
-                Title: "Migration with Outflux",
-                type: PAGE,
-                href: "outflux",
-                children: []
+                Title: "Common Scenarios",
+                type: ANCHOR,
+                href: "common-scenarios",
+                children: [
+                    {
+                        Title: "Get started: Hello, Timescale!",
+                        type: PAGE,
+                        href: "tutorial-hello-timescale",
```

Comment:
So the directory structure uses 'parent' items to determine the page path.  For example, for this entry, the path would be `/tutorials/common-scenarios/tutorial-hello-timescale`.  Unfortunately, this doesn't work for anchors, since you need to add a `#` to the `href` value and adding additional path elements to the end of that doesn't work functionally (or semantically).

The only option I could see would be to add a hidden `DIRECTORY` type items that would have the same href as the anchors, but without the `#` and put the tutorial `PAGE` items as children of those.  The links to those tutorials would have to be updated to reflect that (but they need to be updated anyway)

Reply:
Let me know if this doesn't make sense

Reply:
@solugebefola Maybe we should just accept this as is, then you could do another PR with the above fixes?